youtube_dl/extractor/facebook.py

```diff
@@ -1,17 +1,22 @@
 # coding: utf-8
 from __future__ import unicode_literals
 
+import ast
```

Comment:
looks like we're not using most of the imports here. Please remove the redundant ones